kclvm/tools/src/LSP/src/completion.rs

```diff
@@ -318,6 +318,91 @@ fn completion_dot(
         }
         None => {}
     }
+
+    if let Some(scope) = gs.look_up_scope(pos) {
```

Comment:
Why do we need to get all the symbols in the scope here? I think it is enough to just modify the get_all_attributes part, and the complete function should not be modified.

Reply:
done @He1pa